AI_ML/REGRESSION/LEAST_SQUARES/LEAST_SQUARES.py

```diff
@@ -1,13 +1,14 @@
 import numpy as np
-import traceback
-from flojoy import flojoy, DataContainer
+from typing import Optional
+from flojoy import flojoy, OrderedPair, Matrix, Vector
 
 
 @flojoy
-def LEAST_SQUARES(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
+def LEAST_SQUARES(
+    a: OrderedPair | Matrix | Vector, b: Optional[OrderedPair | Matrix | Vector] = None
+) -> Matrix | Vector:
```

Comment:
Returning vector doesn't make sense either, a regression has to have x and y coordinates